ci/generate-readme.sh

```diff
@@ -2,12 +2,20 @@
 
 set -e
 
+## libelf1 is needed to run flow
+apt-get update -qq
+apt-get install -qy git
+
 yarn install --no-progress
 yarn generate-readme
-git add src/README.md
-if !git diff-index --quiet HEAD -- src/README.md; then
+
+if ! git diff-index --quiet HEAD -- src/README.md
+then
+  git add src/README.md;
   git config --global user.name "nemobot";
   git config --global user.email "our-bots@buildo.io";
   git commit -m "Update README [skip CI]";
   git push origin master;
```

Comment:
@gabro this will probably still fail on `master`, what would be the right way to do it with concourse? We have a GitHub private-key in the secrets, and also a token, maybe I could use those...suggestions?